_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
```

Comment:
Since everything here is related to Kubernetes, do we really need to specify it as *K8s* Objects? Also, I'm unclear if there are different types of objects in Kubernetes. I know that there are API Objects, but are there also other things that are referred to as "objects" in the Kubernetes nomenclature?